charts/gardener/controlplane/local-values.yaml

```diff
@@ -3,6 +3,7 @@
 global:
   # Gardener API server configuration values
   apiserver:
+    clusterIdentity: garden-cluster-identity
```

Comment:
See below - for the `local-values.yaml` it probably doesn't matter too much. Actually, what is the `local-values.yaml` used for? Can it be deleted?

Reply:
I am not sure. I think the idea was to provide a minimal viable values.yaml with some standard configuration. 
But I never used it or introduced it.
I would say it can be deleted.

Reply:
I would delete it in a separate PR. This PR does not touch the existing loca-values.yaml